### DISCUSSION TITLE
Does the app need to be open on my device for it to update my battery?

### DISCUSSION THREAD

**Discussion by neil-ennis** [netzero-labs/netzero#26] [Q&A] [answered]
Please forgive the dumb question.  I'd rather look silly than assume something wrong.

I have the NetZero app installed on my android device.  I also occasionally use the web interface at https://app.netzeroapp.io/

I'd like to make sure everything is working ok before we leave our house for an extended period of time.

Do I need to have the app open for my powerwall to receive price updates, or are the updates sent automatically by a webservice?

I.e.  If I log in once, and add the API key from my power supplier and add a few automations, is that it?  Can I just set it and forget about leaving an app open somewhere?

I'm assuming there's a webservice that does all this, but wasn't sure.

**Comment by zigam** [comment, marked as answer]
No need to keep the app open, automations and dynamic tariffs run on the backend.

**Reply by neil-ennis** [reply]
Many thanks, Zigam.
